tests/Unit/ProtoTests/BasicOneof/out/src/Request/SupplementaryDataOneof.php

```diff
@@ -98,6 +104,8 @@ public function setExtraSummary($extraSummary)
      * Sets this oneof to extra_request and updates its value.
      *
      * @param PartOfRequestA $extraRequest The new value of this oneof.
+     *
+     * @return SupplementaryDataOneof The modified object
      */
     public function setExtraRequest($extraRequest)
```

Comment:
Would we be able to typehint in cases like this where it is feasible?

Reply:
Since we don't typehint in other GAPIC methods (and I'm not sure if our supported versions of PHP for clients would support this), would it be odd or breaking to add it only here? For these reasons, I'd lean towards not typehinting ATM, and doing a major upgrade for all generated methods at some future point.

Reply:
Typehinting against objects is supported in 5.5+. I hadn't realized we don't typehint elsewhere, good point and I agree - let's do it all at once later.